fireant/queries/builder/dataset_blender_query_builder.py

```diff
@@ -204,6 +235,9 @@ def _map_fields(fields):
         orders=[],
         share_dimensions=dataset_share_dimensions,
     )
+    if result[0].get_sql() == "":
+        return []
```

Comment:
Out of interest, what is the case where the sql can be an empty string?

Reply:
I had some cases before but I think with all the later optimisation commits this should no longer happen. I removed it from the code so we can test whether it occurs.